app/models/bulkrax/xml_entry.rb

```diff
@@ -27,7 +27,6 @@ def self.data_for_entry(data, path = nil)
               Nokogiri::XML::Node::SaveOptions::NO_DECLARATION | Nokogiri::XML::Node::SaveOptions::NO_EMPTY_TAGS
           ).delete("\n").delete("\t").squeeze(' '), # Remove newlines, tabs, and extra whitespace
         collection: collections,
-        file: record_file_paths(path),
```

Comment:
this is the one edit I'm least sure about - I think attach every file that's at this path is what this code was doing, which I don't think is right for the XML parser, but I'd love clarification